libs/designsystem/src/lib/components/action-sheet/action-sheet.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { fakeAsync, tick } from '@angular/core/testing';
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import { MockComponent } from 'ng-mocks';
+
+import { PlatformService } from '../..';
+import { OpenState } from '../../models';
+import { WindowRef } from '../../types';
+import { ButtonComponent } from '../button/button.component';
+import { IconComponent } from '../icon';
+import { ModalController } from '../modal';
+
+import { ActionSheetComponent, ActionSheetPopoutComponent } from './';
+
+describe('ActionSheetComponent', () => {
+  let spectator: SpectatorHost<ActionSheetComponent>;
+  let popout;
+  const openDelayInMs = ActionSheetComponent.OPEN_DELAY_IN_MS;
+  const modalControllerSpy = jasmine.createSpyObj('ModalController', ['showActionSheet']);
+  const mockPlatformService = { isTouch: () => false };
+  const createHost = createHostFactory({
+    component: ActionSheetComponent,
+    declarations: [
+      MockComponent(IconComponent),
+      MockComponent(ButtonComponent),
+      MockComponent(ActionSheetPopoutComponent),
+    ],
+    providers: [
+      {
+        provide: WindowRef,
+        useValue: window,
+      },
+      {
+        provide: ModalController,
+        useValue: modalControllerSpy,
+      },
+      {
+        provide: PlatformService,
+        useValue: mockPlatformService,
+      },
+    ],
+  });
+
+  beforeEach(() => {
+    spectator = createHost('<kirby-action-sheet></kirby-action-sheet>');
+
+    spectator.component.header = 'Test header';
+    spectator.component.subheader = 'Test subheader';
+    spectator.component.items = [
+      { id: '1', text: 'Action 1' },
+      { id: '2', text: 'Action 2' },
+      { id: '3', text: 'Action 3' },
+    ];
+    spectator.component.cancelButtonText = 'Test cancel button text';
+    popout = spectator.element.getElementsByTagName('kirby-action-sheet-popout')[0];
```

Comment:
Use spectator.query instead:
```suggestion
    popout = spectator.query('kirby-action-sheet-popout');
```

Reply:
Fixed